file_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"os"
+	"testing"
+	"path/filepath"
+	"io/ioutil"
+)
+
+func TestDownloadZipFile(t *testing.T) {
+	cases := []struct {
+		repoOwner   string
+		repoName    string
+		gitTag      string
+		githubToken string
+	}{
+		{"gruntwork-io", "fetch-test-public", "v0.0.1", ""},
+		{"gruntwork-io", "fetch-test-private", "v0.0.2", os.Getenv("GITHUB_OAUTH_TOKEN")},
+	}
+
+	for _, tc := range cases {
```

Comment:
You'd think I would follow my own `IMPORTANT!` warning...thanks for catching. Added.